contracts/Dependencies/Mynt/MyntLib.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity 0.6.11;
 import "./IMassetManager.sol";
 import "./IDLLR.sol";
 import "../SafeMath.sol";
+import { IPermit2, ISignatureTransfer } from "../../Interfaces/IPermit2.sol";
```

Comment:
The pragma version is different between the interface in contracts/interfaces (0.6.11) & the Dependencies/permit2/interfaces/ (0.8.xx)
We can't use the interface inside Dependencies/permit2/interfaces/ since zero contracts are using 0.6.11

Reply:
can use ABIs from permit2 SDK for deployments in tests. a new refactoring ticket SOV-3769 is created to fix this. 
resolved for now.